unlock-app/src/createUnlockStore.js

```diff
@@ -53,7 +53,7 @@ export const createUnlockStore = (
     provider: providerReducer,
     transactions: transactionsReducer,
     currency: currencyReducer,
-    error: errorReducer,
+    errors: errorReducer,
```

Comment:
Stick to `error` since this is `errorReducer` too or switch to `errorsReducer` (my preference since the point of this PR is to add support for multiple errors in the store)

Reply:
moving to errorsReducer